docs/reference/plugins.rst

```diff
@@ -191,13 +191,20 @@ CMSPluginBase Attributes and Methods Reference
     ..  attribute:: text_enabled
 
         Default: ``False``
+        
+        This attribute controls whether your plugin will be usable (and rendered) 
+        in a text plugin. When you edit a text plugin on a page, the plugin will show up in
+        the "CMS Plugins" dropdown and can be configured and inserted. The output will even
+        be previewed in the text editor.
 
-        Not all plugins are usable in text plugins. If your plugin *is* usable in a text plugin:
+        Of course, not all plugins are usable in text plugins. Therefore the default of this
+        attribute is ``False``. If your plugin *is* usable in a text plugin:
 
         * set this to ``True``
-        * make sure your plugin provides its own :meth:`icon_src`
+        * make sure your plugin provides its own :meth"`icon_alt`, this will be used as a tooltip in 
+        the text-editor and comes in handy when you use multiple plugins in your text.
```

Comment:
@jeroenpeters1986 I think you are missing 2 spaces in front. Maybe that it will be merged.